### ISSUE TITLE
:spacer, but need very precise space

### ISSUE THREAD

**Issue by kwladyka** [yogthos/clj-pdf#87]
```clojure
[:paragraph {:align :center
                :leading 4
                :size 4
                :style :bold
                :encoding :unicode
                :ttf-name "Candara.ttf"
                :color [255 255 255]}
    "African Queen"]
```

I need to have space before that with height equal exactly 1. How to achieve that? I theory i have `[:spacer]` but i can't control the height of this.

**Comment by yogthos** [comment]
Yeah, unfortunately I'm not sure there's a way to handle this. The layout happens automatically and it doesn't look like it allows for precise control. One option could be to use SVG as input as it will be rendered as absolute position on the page.

**Comment by kwladyka** [comment]
I don't know how it is working, but maybe it is good idea to get some ideas from [FPDF](http://www.fpdf.org). There is option to set position.

After closer look on clj-pdf i have too many problems to generate labels. Probably i have to give up using this package, but i will have an eye on that. I hope one day i will change solution for your package :)

**Comment by yogthos** [comment]
It does sound like the library might not be the best fit for what you're doing.

**Comment by kwladyka** [comment]
Yes. Anyway i wrote feedback about library. If you don't plan make changes maybe it will be good idea to write about cons of the library in readme. It can save a lot of time other people.

**Comment by yogthos** [comment]
At some point I'd like to investigate if it's possible to address these issues, so I'll leave them open for the time being as a reference.
